pkg/firewall/firewalld.go

```diff
@@ -113,7 +114,9 @@ func (fb *fwdBackend) Del(conf *FirewallNetConf) error {
 		// Remove firewalld rules which assigned the given source IP to the given zone
 		firewalldObj := fb.conn.Object(firewalldName, firewalldPath)
 		var res string
-		firewalldObj.Call(firewalldZoneInterface+"."+firewalldRemoveSourceMethod, 0, getFirewalldZone(conf), ipStr).Store(&res)
+		if err := firewalldObj.Call(firewalldZoneInterface+"."+firewalldRemoveSourceMethod, 0, getFirewalldZone(conf), ipStr).Store(&res); err != nil {
+			logrus.Errorf("unable to store firewallobj")
```

Comment:
I am not familiar with those parts of the code but firewall sounds security relevant. To be sure: really not return the error?

Reply:
imo, given the error wasnt being caught and it is on a remove, i choose to log it. im open to strong feelings otherwise.